src/main/java/org/sunbird/profile/service/UserBulkUploadService.java

```diff
@@ -141,21 +158,30 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
                         errorDetails = nextRow.createCell(5);
                     }
                     totalRecordsCount++;
-                    if (errList.isEmpty()) {
-                        boolean isUserCreated = userUtilityService.createUser(userRegistration);
-                        if (isUserCreated) {
-                            noOfSuccessfulRecords++;
-                            statusCell.setCellValue(Constants.SUCCESS.toUpperCase());
-                            errorDetails.setCellValue("");
+                    if (!errList.isEmpty()) {
+                        str.append("Failed to Process the Uploaded File. Error Details - [").append(errList).append("]");
```

Comment:
This is not the code I've expected. errList still contains "is missing" for each field. 
Just to make sure the error message is as follows:
Failed to process user record. Missing Parameters - [First Name, Last Name, Email, Phone]